conf/utils.py

```diff
@@ -62,34 +63,41 @@ def get_env(varname, default=None):
             exit(1, "Empty env var {}".format(varname))
 
 
-def ips_by_interface(name):
-    ipdb = IPDB()
-    return [ipobj[0] for ipobj in ipdb.interfaces[name]["ipaddr"].ipv4]
+def ips_by_interface(name) -> list:
```

Comment:
Add parameter typing, and can you be more specific in the list of returns. Ex. List[str].

Reply:
Added.